iroha/src/telemetry.rs

```diff
@@ -0,0 +1,59 @@
+//! Module with telemetry actor
```

Comment:
Please add more description of what this actor does.

Reply:
It should be general purpose thing. That is why i left no comments here